react/FilePicker/FilePickerBodyItem.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import filesize from 'filesize'
+import { makeStyles } from '@material-ui/core/styles'
+
+import { models } from 'cozy-client'
+
+import ListItem from '../MuiCozyTheme/ListItem'
+import ListItemIcon from '../MuiCozyTheme/ListItemIcon'
+import ListItemText from '../ListItemText'
+import Divider from '../MuiCozyTheme/Divider'
+import Icon from '../Icon'
+import FileTypeText from '../Icons/FileTypeText'
+import FileTypeFolder from '../Icons/FileTypeFolder'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+import { useI18n } from '../I18n'
+
+const {
+  file: { isDirectory, isFile }
+} = models
+
+const useStyles = makeStyles(() => ({
+  verticalDivider: {
+    height: '2rem',
+    display: 'flex',
+    alignSelf: 'auto',
+    alignItems: 'center',
+    marginLeft: '0.5rem'
+  },
+  listItemIcon: {
+    marginLeft: '1rem'
+  }
+}))
+
+const styleWrapper = { display: 'contents' }
```

Comment:
Be careful, `display:contents` is not supported by all of our targeted browsers
https://caniuse.com/css-display-contents 

Specially for Safari and old version of Android. 

I found that https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/filelist/fileopener.styl#L1-L10 maybe it can help you